### ISSUE TITLE
jax.profiler should suggest that installation of tensorflow is necessary

### ISSUE THREAD

**Issue by annulen** [google/jax#16577]
Right now attempt to use `jax.profiler` without tensorflow results in diagnostic messages from XLA like
```
E external/xla/xla/python/profiler/internal/python_hooks.cc:398] Can't import tensorflow.python.profiler.trace
```
It would be better if `jax.profiler` had an explicit suggestion that user should install `tensorflow` package manually, and dependency on Tensorflow was mentioned in documentation (at https://jax.readthedocs.io/en/latest/profiling.html and https://jax.readthedocs.io/en/latest/_autosummary/jax.profiler.trace.html)

**Comment by mcobzarenco** [comment]
Is tensorflow required to trace a jax program?

**Comment by KaleabTessera** [comment]
Yep, tf is required - https://jax.readthedocs.io/en/latest/profiling.html#installation .
